spyder_kernels/console/kernel.py

```diff
@@ -97,7 +97,6 @@ def __init__(self, *args, **kwargs):
 
         self.namespace_view_settings = {}
         self._mpl_backend_error = None
-        self._running_namespace = None
```

Comment:
The namespace are now organised as a stack, as you can technically call runfile in a file